python/common/org/python/types/Int.java

```diff
@@ -45,7 +49,7 @@ public int hashCode() {
         }
     }
 
-    private Int(long value) {
+    public Int(long value) {
```

Comment:
This constructor is private for an important reason - you shouldn't be instantiating integers directly, because it undermines instance caching.

Reply:
I switched everything over to `Int.getInt()`, I hope.